src/testdir/test_cmdline.vim

```diff
@@ -293,14 +293,13 @@ func Test_changing_cmdheight()
   " :resize now also changes 'cmdheight' accordingly
   call term_sendkeys(buf, ":set cmdheight+=1\<CR>")
   call VerifyScreenDump(buf, 'Test_changing_cmdheight_2', {})
-  call term_sendkeys(buf, ":set cmdheight-=1\<CR>")
```

Comment:
I added this in #16359 to ensure the following tests didn't need a change but missed that this made dump 2 and 3 identical.